backend/models/user.py

```diff
@@ -9,23 +9,37 @@ class User(db.Model):
         db.Integer,
         primary_key=True)
 
+    sub = db.Column(
+        "sub",
+        db.String(),
+        unique=True,
+    )
+
     username = db.Column(
         "username",
         db.String(30),
         unique=True)
 
+    email = db.Column(
+        "email",
+        db.String(30),
```

Comment:
thomas.alejandro.ramirez.fernandez@online.ntnu.no will be too long for this, could it be _a lot_ bigger?